src/dscanner/analysis/static_if_else.d

```diff
@@ -47,17 +48,7 @@ class StaticIfElse : BaseAnalyzer
 
 	const(IfStatement) getIfStatement(const ConditionalStatement cc)
 	{
-		if (cc.falseStatement.statement)
-		{
-			if (cc.falseStatement.statement.statementNoCaseNoDefault)
-			{
-				if (cc.falseStatement.statement.statementNoCaseNoDefault.ifStatement)
-				{
-					return cc.falseStatement.statement.statementNoCaseNoDefault.ifStatement;
-				}
-			}
-		}
-		return null;
+		return safeAccess(cc).falseStatement.statement.statementNoCaseNoDefault.ifStatement;
```

Comment:
This change is :1st_place_medal:

Reply:
Another bonus point for UFCS: `cc.safeAccess.falseStatement....` ;-)